Sources/AnimatedImageView.swift

```diff
@@ -305,6 +305,7 @@ class Animator {
     fileprivate let maxTimeStep: TimeInterval = 1.0
     fileprivate var frameCount = 0
     fileprivate var currentFrameIndex = 0
+    fileprivate var currentFrameIndexInBuffer = 0
```

Comment:
`currentFrameIndex` now tracks the current frame of the gif being displayed. `currentFrameIndexInBuffer` serves the old purpose of `currentFrameIndex`, which is the index of the current frame of the gif as it exists in the `animatedFrames` buffer.